docs/specification/appendix.md

```diff
@@ -67,6 +71,12 @@ Schema `encoding` may only be omitted for self-describing message encodings such
 - `encoding`: `flatbuffer`
 - `data`: A [reflection.Schema](https://github.com/google/flatbuffers/blob/master/reflection/reflection.fbs) flatbuffer describing the parsed schema encoded as a binary flatbuffer. This can be generated by running `flatc -b --schema` on the fbs file.
 
+### capnproto
+
+- `name`: Fully qualified name: `[schema display name].Name`. For example, the struct `Bar` in schema file with path `path/to/files/schema.capnp` has the default qualified name `path/to/files/schema.capnp.Bar`. `[schema display name]` can be overridden in the `displayName` field of each schema file.
+- `encoding`: `capnproto`
+- `data`: The target `Schemas` and their dependencies serialized into a byte stream in the following format `[ schema 0 | schema 1 | ... | schema n ]`, via the method [`Schema::asUncheckedMessage`](https://github.com/capnproto/capnproto/blob/b2fdf71399accdf4ab93d38421a3225440b1d108/c%2B%2B/src/capnp/schema.h#L91). The schemas do not need to be an any particular order. TODO: This can be generated by running e.g. `mcap dump-capnp-schema --files my_schemas/*.capnp [--unqualified-names] > my_schemas.capnp.schema`. `--unqualified-names` will cause e.g. the `name` for `Bar` in `my_schemas/foo.capnp` to be `foo.Bar`. Note that this will fail for duplicate file names (we will validate duplication of filenames).
```

Comment:
Hm, I wonder if we should use the CodeGeneratorRequest format instead, described here: https://github.com/capnproto/capnproto/issues/673#issuecomment-388994957

Reply:
Ooh, great catch! Will look into that

Reply:
Updated to use it, seems to work. PTAL.